### PR TITLE
[Site] UX Map page

### LINKED ISSUE #2042: Add the Map component to the UX website
Symfony UX has a new Map component: https://github.com/symfony/ux/pull/1937

Hugo (@Kocal) is still tweaking and polishing some of its features, but when the component stabilizes, we need to add it to:

* The list of packages: https://ux.symfony.com/packages
* Create a demo for it in: https://ux.symfony.com/demos

Thanks!

### PR DESCRIPTION
**Pull request by smnandre**
(very simple page for now, but i'd like to deploy _a_ page asap... and then we'll have more time for demos, docs...)

ux-map-screen

⚠️  Will need rebase / composer after 2.19.2 (and some check on the controller.json)

Closes #2042

### CODE REVIEW

File: ux.symfony.com/composer.json [hunk 1]
```diff
@@ -38,7 +38,9 @@
         "symfony/ux-dropzone": "2.x-dev",
         "symfony/ux-icons": "2.x-dev",
         "symfony/ux-lazy-image": "2.x-dev",
+        "symfony/ux-leaflet-map": "^2.19",
```
Comment (Kocal) [inline ux.symfony.com/composer.json]: Why not using `2.x-dev` like for other ux packages?

File: ux.symfony.com/src/Controller/UxPackage/MapController.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        $map = (new Map('default'))
+            ->center(new Point(45.7534031, 4.8295061))
+            ->zoom(6)
+
+            ->addMarker(new Marker(
+                position: new Point(45.7534031, 4.8295061),
+                title: 'Lyon',
+                infoWindow: new InfoWindow(
+                    content: '<p>Thank you <a href="https://github.com/kocal">@kocal</a> for this component!</p>',
```
Comment (Kocal) [inline ux.symfony.com/src/Controller/UxPackage/MapController.php]:
```suggestion
                    content: '<p>Thank you <a href="https://github.com/Kocal">@Kocal</a> for this component!</p>',
```

🙏🏻

**Review by Kocal — APPROVED** [review]

File: ux.symfony.com/composer.json (hunk 1 above)
Comment (smnandre) [reply ux.symfony.com/composer.json]: Mistake.. 😅  thx

**Review by Kocal — APPROVED** [review]
Thanks Simon!

**Comment by kbond** [comment]
Thanks Simon.

**Pull request merged**